honeybee_energy/properties/room.py

```diff
@@ -953,22 +966,23 @@ def solve_norm_perimeter_flow_coefficient(norm_area_flow_coefficient, face_area,
         not normalized. The normalized perimeter air mass flow coefficient is derived
         from its infiltration flow rate using the following formula::
 
-            Qv * d * A = Cql * L * dP^n
+            Qva * d * A = Cql * L * dP^n
 
             where:
                 Cql: Air mass flow coefficient per unit length at 1 Pa [kg/m/s/P^n]
-                Qv: Volumetric air flow rate per length [m3/s/m]
+                Qva: Volumetric air flow rate per length [m3/s/m]
                 d: Air density [kg/m3]
                 A: Surface area of opening [m2]
                 L: Surface perimeter of opening [m]
                 dP: Change in pressure across building envelope [Pa]
                 n: Air mass flow exponent [-]
 
-        Since ``Qv * d / dP^n`` equals ``Cqa`` the normalized area flow coefficient,
+        Since ``(Qva * d) / dP^n`` equals ``Cqa`` the normalized area flow coefficient,
         this can be simplified and rearranged to solve for ``Cql`` with the following
-        formula::
-
-            Cql = Cqa * A / L
+        formula:
```

Comment:
We need the extra line break and the `::` here to make sure that the formula displays correctly in the online docs like so:

https://www.ladybug.tools/honeybee-energy/docs/honeybee_energy.properties.room.html#honeybee_energy.properties.room.RoomEnergyProperties.solve_norm_perimeter_flow_coefficient